specification/columns/contractappliedquantity.md

```diff
@@ -0,0 +1,40 @@
+# Contract Applied Quantity
```

Comment:
Is this intended to cover a scenario where someone commits to a specific quantity of usage, rather than cost? Is this required that's not the case? Is ContractAppliedCost required when quantity is committed? I'm not sure how a quantity can be applied to a cost commitment, so this feels confusing unless it's null when not applicable. Same for the ContractAppliedCost column when there's a quantity commitment.